packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.stories.js

```diff
@@ -123,7 +122,50 @@ storiesOf('Tasks', module)
           step={step}
           subjectReadyState={subjectReadyState}
           tasks={tasks}
-          theme={dark ? 'dark' : 'light'}
+        />
+      </Grommet>
+    </Provider>
+  )
+})
+.add('text', function () {
+  const step = null
+  const tasks = [
+    {
+      annotation: { task: 'T0', value: '' },
+      help: 'Type something into the text box.',
+      instruction: 'Type something here',
+      taskKey: 'T0',
+      text_tags: ['insertion', 'deletion'],
+      type: 'text',
+      updateAnnotation: sinon.stub()
+    }
+  ]
+  const dark = boolean('Dark theme', false)
+  const loadingState = select('Subject loading', asyncStates, asyncStates.success)
+  const subjectReadyState = select('Subject loading', asyncStates, asyncStates.success)
+  const store = Object.assign({}, mockStore, {
+    subjectViewer: {
+      loadingState
+    },
+    workflows: {
+      loadingState: asyncStates.success
+    },
+    workflowSteps: {
+      activeStepTasks: tasks,
+      isThereANextStep: () => false,
+      isThereAPreviousStep: () => false,
+      isThereTaskHelp: true
+    }
+  })
+  return (
+    <Provider classifierStore={store}>
+      <Grommet theme={Object.assign({}, zooTheme, { dark })}>
+        <Tasks
+          isThereTaskHelp={true}
```

Comment:
Non-blocking, maybe make this toggleable using a knob?

Reply:
I've added a bunch of improvements to the story styling, including this. Slightly outside the scope of this PR but all the Tasks stories should be looking better now.